lib/aes.ex

```diff
@@ -26,26 +26,17 @@ defmodule Fields.AES do
       iex> is_binary(ciphertext)
       true
   """
-  @spec encrypt(any) :: String.t()
-  def encrypt(plaintext) do
-    # create random Initialisation Vector
-    iv = :crypto.strong_rand_bytes(16)
-    # get the *latest* key in the list of encryption keys
-    key = get_key()
-    {ciphertext, tag} = :crypto.block_encrypt(:aes_gcm, key, iv, {@aad, to_string(plaintext), 16})
-    # "return" iv with the cipher tag & ciphertext
-    iv <> tag <> ciphertext
-  end
 
   @spec encrypt(any, number) :: {String.t(), number}
-  def encrypt(plaintext, key_id) do
+  def encrypt(plaintext, key_id \\ get_key_id()) do
```

Comment:
if the `key_id` argument is not passed, then `get_key_id` will be the default key_id ie the last index of the list

Reply:
@SimonLab we can probably simplify this by only having `encrypt/1` because the "_user_" of fields does not need to _think_ about which encryption key the are using. Provided they have the `ENCRYPTION_KEYS` environment variable defined, they should never need to _think_ about keys again.

Reply:
Yes agree, I'll update the test to only use `encrypt/1`